test/functional/sqlsrv/sqlsrv_buffered.phpt

```diff
@@ -12,9 +12,9 @@ function insertOneRow($conn, $tableName)
     $result = null;
     if (AE\isColEncrypted()) {
         $data = array("Field2" => "This is field 2.",
-                      "Field3" => array("010203", null, SQLSRV_PHPTYPE_STRING(SQLSRV_ENC_CHAR), SQLSRV_SQLTYPE_IMAGE),
+                      "Field3" => "010203",
                       "Field4" => "This is field 4.",
-                      "Field5" => array("040506", null, SQLSRV_PHPTYPE_STRING(SQLSRV_ENC_CHAR), SQLSRV_SQLTYPE_VARBINARY('max')),
+                      "Field5" => "040506",
```

Comment:
Why have these values been changed?

Reply:
They were the workarounds added while refactoring the tests to bind input parameters. With this fix the workarounds are no longer needed.